corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -124,13 +124,14 @@ <h3 class="panel-title">
       <thead>
         <tr>
           <!-- ko foreach: columns -->
-          <th data-bind="style: { width: width() + 'px' }">
-            <span data-bind="text: title"></span>
-            <button type="button"
-                    class="btn btn-default"
-                    data-bind="click: $root.editColumnController.set">
-              {% trans "Edit" %}
-            </button>
+          <th data-bind="style: { width: width() + 'px' },
+                         columnResize: width">
+            <a class="datagrid-edit-link"
+               href="#"
```

Comment:
Can this be blank? We've had problems with `#` hrefs messing with tab/nav behavior.

Reply:
done